src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -68,17 +68,17 @@ namespace AppInstaller::CLI::Workflow
         case ManifestInstaller::InstallerTypeEnum::Msi:
         case ManifestInstaller::InstallerTypeEnum::Nullsoft:
         case ManifestInstaller::InstallerTypeEnum::Wix:
-            context << DownloadInstallerFile << VerifyInstallerHash;
+            context << DownloadInstallerFile << VerifyInstallerHash << UpdateInstallerFileMotwIfApplicable;

```

Comment:
>UpdateInstallerFileMotwIfApplicable [](start = 71, length = 35)

Why not move VerifyInstallerHash down, and make MSStore return instead of break?  I don't think even more duplication is really the right answer.

Also, does this mean that Store installs were broken?

Reply:
Store install is not broken. The first line in UpdateInstallerFileMotwIfApplicable checks if there's a downloaded file, so it's technically skipped in store scenario.

I moved up just to be more explicit that each step is applicable to which type. I think moving down and let individual type (i.e. msstore) opt-out by using return is more likely to be overlooked in the future;

---
In reply to: [514492065](https://github.com/microsoft/winget-cli/pull/627#discussion_r514492065) [](ancestors = 514492065)